tests/viscoelastic_fixed_elastic_time_step.prm

```diff
@@ -0,0 +1,132 @@
+#  Global parameters
+set Dimension                              = 2
+set Start time                             = 0
+set End time                               = 1e3
+set Use years in output instead of seconds = true
+set Linear solver tolerance                = 1e-7
+set Nonlinear solver scheme                = IMPES
```

Comment:
Why do you use IMPES for your test cases, isn't the problem nonlinear?

Reply:
In this case I believe the problem is linear as there is no plasticity or non-linear viscous flow terms. Would one need to use the iterative solver simply due to having the stress (strain-rate dependent) in the constitutive law?

Reply:
I guess I never thought about that, as models with elasticity, but without plasticity don't come up very often. But I think as soon as your viscosity depends on the solution variables of the Stokes system, the problem is nonlinear. 
You can easily test this by just using the iterated IMPES solver, and if the nonlinear solver converges after the second iteration with a very small residual, the problem is linear; otherwise you probably need the nonlinear solver.